bulbs/content/tasks.py

```diff
@@ -199,7 +201,9 @@ def post_to_instant_articles_api(content_pk):
                 source,
                 fb_page_id,
                 fb_api_url,
-                fb_token_path)
+                fb_token_path,
+                fb_dev_mode,
```

Comment:
Ah, yeah that's true. I think so, because we don't want to send every article to development mode, we just want to use this for testing purposes. I think it also makes sense as an additional safety measure, but I'm fine with removing it as well.